README.md

```diff
@@ -0,0 +1,22 @@
+# INF 7420 Project - Wireframe
+For my project, I am going to create a professional website that will include my e-Portfolio. I was planning on creating one anyone, once I graduated, but this way I get to create something that will also be included in my e-Portfolio.
+## Wireframes
+I have included wireframes for my home, meditations, and CV pages.
+### **Home Page**
+![Wireframe of Home Page](HomePage.jpg)
+*Header*: will include my name in a cursive font and a fixed navigation menu.
+*Navigation Menu*: includes links to the main pages- home, e-Porfolio, works, CV, meditations, and contact information.
+*Main Content*: will include a brief but thorough "about me" section and a small image to the left.
+*Footer*: will include links to social media, privacy policy and copyright information.
+### **CV Page**
+![Wireframe of CV Page](CVPage.jpg)
```

Comment:
The source of the image files is in the Wireframes directory. Please update this to include the full path: `Wireframes/CVPage.jpg`